Actions/CalculateArtifactNames/action.yaml

```diff
@@ -21,6 +18,9 @@ inputs:
     description: Suffix to add to the artifacts names
     required: false
     default: ''
+  settingsJson:
+    description: 'OBSOLETE: Settings from repository in compressed Json format'
```

Comment:
Could we not just remove the parameter if it's not used anymore? I know it would break people running in preview (once we deploy to preview) but aren't we breaking them anyway by removing the SettingsJson output from ReadSettings?

Reply:
We should really find a way, where Update AL-Go System Files doesn't call the "old" update code, but the new - this will make things much easier for us.
I will think about how this can be done - but not for this PR.

Reply:
Ahh right.. Yeah that wouldn't be good. But then we could remove it from all other actions then and keep it in ReadSecrets as the only place?

Reply:
We could - but this PR in itself doesn't actually break pipelines - even though it is fairly big. 
This is why I waited and in the action I would display that they are running an old version of AL-Go - they should update.